core/arch/arm/kernel/secure_partition.c

```diff
@@ -352,6 +380,22 @@ static TEE_Result sp_dt_get_u32(const void *fdt, int node, const char *property,
 	return TEE_SUCCESS;
 }
 
+static TEE_Result sp_dt_get_u16(const void *fdt, int node, const char *property,
+				uint16_t *value)
+{
+	uint32_t dt_value;
```

Comment:
Please initialize the variable.
I would have picked a shorter name like `v` for instance, it's obvious what it's for anyway.